app/user/serializers.py

```diff
@@ -0,0 +1,71 @@
+from typing import Any
+
+from django.contrib.auth import get_user_model
+from django.core.mail import send_mail
+from django.template.loader import render_to_string
+from rest_framework import serializers
+from rest_framework.validators import UniqueValidator
+
+from speaksfer.settings.base import EMAIL_USER
+
+User = get_user_model()
+Profile = get_user_model()
```

Comment:
No only the user model is gotten this way, other models are gottebn the normal way using relative imports